manim/utils/file_ops.py

```diff
@@ -23,6 +25,12 @@ def add_extension_if_not_present(file_name, extension):
         return file_name
 
 
+def add_version_before_extension(file_name):
+    file_name = Path(file_name)
+    path, name, suffix = file_name.parent, file_name.stem, file_name.suffix
+    return Path(f"{path}\\{name}_ManimCE_v{__version__}{suffix}")
```

Comment:
This is my mistake, I should've told you to use the `.name` attribute and not the `.stem` attribute
```suggestion
    name, suffix = file_name.name, file_name.suffix
    return Path(f"{name}_ManimCE_v{__version__}{suffix}")
```

Reply:
Names it wrong and places it in the wrong directory. I think I'd tried that before.

Reply:
Ah, yeah, sorry. Could you turn the backslashes into a single forward slash though?

Reply:
If I remember correctly, pathlib returns a path with 2 backslashes. I'd rather stay consistent.

Reply:
Are you on Windows? That would make more sense then. But one of the appeals of pathlib is that it automatically handles path separators for you, and I certainly think `/` is better than `\\`, and is in line with the `/` operator used elsewhere in manim and this PR, and is the path separator used for basically all platforms besides Windows